source/documentation/services/domainmgt.html.md.erb

```diff
@@ -29,3 +29,5 @@ There is no charge for any of the above services. Costs are met centrally by Pla
 ## Support
 
 If you require any of these services please contact #ask-operations-engineering slack channel or [domains@digital.justice.gov.uk](mailto:domains@digital.justice.gov.uk).
+
+Alternativly for Ministry of Justice Service Teams you can raise a PR for changes directly via [DNS](https://github.com/ministryofjustice/dns) repositiry.
```

Comment:
/suggestion
Alternatively, for Ministry of Justice Service Teams, you can raise a PR for changes directly via [DNS](https://github.com/ministryofjustice/dns) repository.

Reply:
Update.